.appveyor.yml

```diff
@@ -5,25 +5,9 @@ environment:
     PYTHON: 'C:\Python27\python.exe'
   - GENERATOR: 'Visual Studio 10 Win64'
     PYTHON: 'C:\Python27-x64\python.exe'
-  - GENERATOR: 'Visual Studio 10'
```

Comment:
I intend to remove these modifications after https://github.com/sdg-mit/gitless/pull/158